### DISCUSSION TITLE
Theme takes precedence over component properties.

### DISCUSSION THREAD

**Discussion by IagoLast** [chakra-ui/chakra-ui#8046] [Q&A] [answered]
# [👉 Link to a code sandbox](https://codesandbox.io/s/prod-hill-vj64cs?file=/src/index.js)

## Description 

The default checkbox component allows you to style the label color using the color property:

```ts
<Checkbox color="blue">
        This text should be blue
</Checkbox>
```
<img width="235" alt="Screenshot 2023-10-03 at 13 05 56" src="https://github.com/chakra-ui/chakra-ui/assets/2657897/c0f30e96-14d8-4922-9ee1-d933a3dbf02f">

However if we create a theme and override the checkbox properties, this takes precedence over the attributes.

```tsx
const theme = extendTheme({
  components: {
    // Override the checkbox component on the theme
    Checkbox: defineMultiStyleConfig({
      sizes: {
        // On the md size we put a red color on the label
        md: definePartsStyle({
          label: { color: "red" }
        })
      }
    })
  }
});
```

```tsx
function App() {
  return (
    <ThemeProvider theme={theme}>
      {/* We expect the color to be blue :/ */}
      <Checkbox size="md" color="blue">
        This text should be blue
      </Checkbox>
    </ThemeProvider>
  );
}
```

Even if we have the `color="blue` the resulting label is red:

<img width="248" alt="Screenshot 2023-10-03 at 13 07 21" src="https://github.com/chakra-ui/chakra-ui/assets/2657897/cb808095-0dec-4bb2-99cc-5f156d4d6c84">

Should I report a bug? Is this expected? Am I doing something wrong?

**Comment by segunadebayo** [comment, marked as answer]
This is a bug in Chakra. The issue here is that the color prop isn't passed down to the label part in the code
https://github.com/chakra-ui/chakra-ui/blob/63431c1ef3b7e8ac9b97c94ca025dd54b9aa3338/packages/components/checkbox/src/checkbox.tsx#L161

Feel free to open a PR

**Reply by IagoLast** [reply]
Great, just created [this PR.](https://github.com/chakra-ui/chakra-ui/pull/8050).

It is my first time contributing here so it might be totally wrong. Please have patience with me :)
